quesma/model/bucket_aggregations/ip_prefix.go

```diff
@@ -0,0 +1,204 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package bucket_aggregations
+
+import (
+	"context"
+	"fmt"
+	"quesma/logger"
+	"quesma/model"
+	"reflect"
+)
+
+// Current limitation: we expect Clickhouse field to be IPv4 (and not IPv6)
+
+// Clickhouse table to test SQLs:
+// CREATE TABLE __quesma_table_name (clientip IPv4) ENGINE=Log
+// INSERT INTO __quesma_table_name VALUES ('0.0.0.0'), ('5.5.5.5'), ('90.180.90.180'), ('128.200.0.8'),  ('192.168.1.67'), ('222.168.22.67')
+
+// TODO make part of QueryType interface and implement for all aggregations
+// TODO add bad requests to tests
+// Doing so will ensure we see 100% of what we're interested in in our logs (now we see ~95%)
```

Comment:
FIY: Probably it make sense to log it once every few minutes. This could be very noisy.